src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1706,6 +1712,20 @@ export default class GeFormRenderer extends LightningElement{
         this.formState = Object.assign({}, this.formState);
     }
 
+    removeFieldsNotUpdatableInFormState(fieldsToUpdate) {
+        FORM_STATE_IMMUTABLE_FIELDS_BY_APINAME.forEach(immutableField => {
+            if (this.isFormStateFieldNotUpdatable(fieldsToUpdate, immutableField)) {
+                delete fieldsToUpdate[immutableField];
+            }
+        });
+
+        return fieldsToUpdate;
+    }
+
+    isFormStateFieldNotUpdatable(fields, field) {
+        return fields.hasOwnProperty(field) && this.formState[field];
```

Comment:
Just to confirm, the second condition `this.formState[field]` means that we _would_ allow a field, defined as immutable in the `FORM_STATE_IMMUTABLE_FIELDS_BY_API_NAME` constant, to be added to formState _if_ the field hasn't been added to formState prior to this form state update attempt?

`isFormStateFieldNotUpdatable` would return false if the field in question is not in formState right and so would not be removed from `fieldsToUpdate`?

Reply:
Your understanding here is 100% correct @psadie-sfdo. We want to make sure that the field value can be set to a non-null value once in form state but block any subsequent update -- until form state is re-initialized.

Reply:
Ahhh, gotcha gotcha. Need to allow that first one when the form initializes. Makes sense and thanks! Deploying the changes to a scratch org now!